### ISSUE TITLE
Cannot update data of type Object

### ISSUE THREAD

**Issue by willianba** [kitsonk/kview#13]
Hey @kitsonk!

After I recently updated to Deno 2 and also updated my kview installation, I no longer see the "Update value" button in any entries in my KV.
I thought this was removed, but I see this is still in kview docs, so maybe I'm doing something wrong.

Idk what else I can share though. It's a simple Object. Here's a screenshot.

![image](https://github.com/user-attachments/assets/917b0f18-d481-49a4-bc79-0133142032eb).

Do you have any guidance or is it a bug?
Thanks!

**Comment by kitsonk** [comment]
When the value contains something that cannot be serialized to JSON, it prevents you from editing it (ref the note in the docs: https://kview.deno.dev/docs/values/objects) because it would not be able to be stored back in the way it was created. I should have made it clearer in the release notes.

What would be better in the long term is an object/array/Map/Set editor that supports set properties and elements in more complex types.

Is the `payment.date` a `Date()` object?

**Comment by willianba** [comment]
> Is the payment.date a Date() object?

yes. i'm thinking about adding a date library to the app, like date-fns. it will probably change this type.

if you have any suggestion on a better way to store dates too, i'm down.

**Comment by kitsonk** [comment]
No, it is a good way. Previous versions of kview would have let you edit is as JSON but then would have destructively changed the value for anything that couldn't be serialized to JSON. If the value doesn't contain anything that isn't serializable to JSON it will let you edit it as JSON.

I started work on an object editor that would allow supporting everything that Deno KV supports and be able to edit individual properties that contain these "complex" values, but it might take me a while.  It will be a bit like Firebase Firestore editor.

**Comment by willianba** [comment]
that's great to know! in the meanwhile, should i close this issue? 😬 
i don't think there's anything i could do to fix that.

**Comment by kitsonk** [comment]
Naw, when I get the object editor done, I will tie it to this issue.
